test/compiler/e2e.ts

```diff
@@ -107,7 +107,7 @@ describe('E2E: Compiler integration', () => {
         const MyLibrary = require(libraryPath);
         const LibraryConsumer = deepCopy(require(resolve(join(configuration.targetPath, 'Two.json'))));
         const myLibrary = await deployContract(wallet, MyLibrary, []);
-        const libraryName = `${configuration.sourcesPath.slice(2)}/MyLibrary.sol:MyLibrary`;
+        const libraryName = `${join(process.cwd(), configuration.sourcesPath.slice(2))}/MyLibrary.sol:MyLibrary`;
```

Comment:
#breakingchange This is possibly breaking change. That would imply, that linking a library now requires an absolute path, which would be most of the time impossible to do, as people link libraries on different machines.